process/smartContract/hooks/blockChainHook.go

```diff
@@ -352,6 +351,27 @@ func (bh *BlockChainHookImpl) IsSmartContract(address []byte) bool {
 	return core.IsSmartContractAddress(address)
 }
 
+// IsPayable checks whether the provided address can receive ERD or not
+func (bh *BlockChainHookImpl) IsPayable(address []byte) (bool, error) {
+	if !bh.IsSmartContract(address) {
+		return true, nil
+	}
+
+	userAcc, err := bh.GetUserAccount(address)
+	if err == state.ErrAccNotFound {
+		return false, nil
+	}
+	if err != nil {
+		return false, err
+	}
+	if check.IfNil(userAcc) {
+		return true, nil
```

Comment:
Smart Contract does not exist (at least not in the current shard), therefore we assume it's payable. Is this ok?

Reply:
resolved. It is false in the newest code